common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -612,3 +625,33 @@ private[comet] case class ConfigBuilder(key: String) {
 private object ConfigEntry {
   val UNDEFINED = "<undefined>"
 }
+
+/**
+ * Utility for generating markdown documentation from the configs.
+ */
```

Comment:
Would you like to put the usage (i.e., `mvn clean package -DskipTests`) in the code comment here?

Reply:
Thanks. Added.